src/lab13.py

```diff
@@ -109,9 +109,6 @@ def needs_compositing(self):
         return any([child.needs_compositing() \
             for child in self.children])
 
-    def clone(self, children):
```

Comment:
Why remove this?

Reply:
Oh—it doesn't matter either way. In Python it's just a runtime error to call an undefined method; adding `assert False` is only worth it if you write a better error message than the default, and we didn't. So I removed this code block from the chapter (I kept the text, just removed the code block) and then for symmetry removed it from the code as well.